examples/alhasan_work_area/duplicate_element_filter.py

```diff
@@ -0,0 +1,13 @@
+def remove_duplicate_filter(lst):
+    modified_list = []
+    for nums in lst:
+        if nums in modified_list:
+            modified_list.remove(nums)
+            modified_list.append(nums)
+        else:
+            modified_list.append(nums)
+    return modified_list
+
+print(remove_duplicate_filter(["ali","ali","alhasan"]))
```

Comment:
According to PEP-8 which also specifies the coding standards, we should put a space after comma , when defining a list